includes/wikia/helpers/FileNamespaceSanitizeHelper.php

```diff
@@ -39,9 +39,15 @@ private function getFilePrefixRegex( $contLang ) {
 			}
 			
 			//be able to match user-provided file namespaces that may contain both underscores and spaces
-			$fileNamespaces = array_map(function( $namespace ) {
-				return mb_ereg_replace('_', '(_|\ )', $namespace);
-			}, $fileNamespaces);
+			$fileNamespaces = array_map( function( $namespace ) {
+				return mb_ereg_replace( '_', '(_|\ )', $namespace );
+			}, $fileNamespaces );
+
+			//be able to match both upper- and lowercase first letters of the namespace
+			foreach ( $fileNamespaces as $namespace ) {
```

Comment:
Changing array you are iterating seems unpredictable. Could you prepare it separately and the merge them? Additionally it seems like perfect array_map use case.